CHANGELOG_PENDING.md

```diff
@@ -1,5 +1,6 @@
 ### SDK Features
 
 ### SDK Enhancements
+* `aws`: Add `WithUseFIPSEndpoint` to `aws.Config`. ([#5078](https://github.com/aws/aws-sdk-go/pull/5078))
```

Comment:
```suggestion
* `aws`: Add `WithUseFIPSEndpoint` to `aws.Config`. ([#5078](https://github.com/aws/aws-sdk-go/pull/5078))
  * `WithUseFIPSEndpoint` can be used to explicitly enable or disable FIPS endpoint variants.
```

Our changelog formatter is unnecessarily specific on what it requires, adding this so it won't break.